src/Common/Tests/TestCases/Failing.cs

```diff
@@ -16,7 +16,9 @@ public class Failing : TestCase
 
         public override void CheckIfAreEqual(object expectedObj, object actualObj)
         {
+#pragma warning disable NUnit2007
```

Comment:
FYI: This was complaining that the expected should not be a constant. I think it this case this could be ignored